.github/workflows/sdk-assets-data-working_with_mltable.yml

```diff
@@ -39,9 +39,9 @@ jobs:
       continue-on-error: true
     - name: run assets/data/working_with_mltable.ipynb
       run: |
-          sed -i -E -e  "s/<SUBSCRIPTION(\s|_)ID>/6560575d-fa06-4e7d-95fb-f962e74efd7a/g" working_with_mltable.ipynb
-          sed -i -E -e "s/<RESOURCE(\s|_)GROUP(\s|_)?(NAME)?>/azureml-examples/g" working_with_mltable.ipynb
-          sed -i -E -e "s/<(AML)?(\s|_)?WORKSPACE(\s|_)NAME>/main/g" working_with_mltable.ipynb
+          sed -i -e "s/<SUBSCRIPTION_ID>/6560575d-fa06-4e7d-95fb-f962e74efd7a/g" working_with_mltable.ipynb
```

Comment:
Are these expected changes or do you think we need to revert these?

Reply:
These changes were not done by me !! I just accepted the incoming changes while resolving the conflicts.
All changes done by me for preprocessing is inside 1h_automl_pipeline folders.

Rest were just taken from main branch to resolve the conflicts